modules/ROOT/pages/salesforce/salesforce-composite-connector-reference.adoc

```diff
@@ -1,889 +1,1187 @@
+| Reconnection a| <<Reconnection>> |  When an app is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+| Host a| String |  Hostname of the proxy. If this property is not set, then no proxy is used, otherwise a proxy is used, but a proxy host must be specified. |  | 
+| Port a| Number |  Port of the proxy. If host is set then this property must be set and cannot be a negative number. |  | 
+| Username a| String |  Username used to authenticate against the proxy. If this property is not set, then no authentication is going to use against the proxy, otherwise this value must be specified. |  | 
+| Password a| String |  Password used to authenticate against the proxy. |  | 
 |===
 
 == Supported Operations
 
-* Create SObject Tree
-* Execute Composite Batch
-* Post Create
-* Post Delete
-* Post Get Limits
-* Post Query
-* Post Query All
-* Post Retrieve
-* Post Search
-* Post Update
-* Pre-Create
-* Pre-Delete
-* Pre-Get Limits
-* Pre-Query
-* Pre-Query All
-* Pre-Retrieve
-* Pre-Search
-* Pre-Update
+* <<createSobjectCollections>> 
+* <<createSobjectTree>> 
+* <<deleteSobjectCollections>> 
+* <<executeCompositeBatch>> 
+* <<executeCompositeRequest>> 
+* <<getSobjectCollections>> 
+* <<postCreate>> 
+* <<postDelete>> 
+* <<postGetLimits>> 
+* <<postQuery>> 
+* <<postQueryAll>> 
+* <<postRetrieve>> 
+* <<postSearch>> 
+* <<postUpdate>> 
+* <<preCreate>> 
+* <<preDelete>> 
+* <<preGetLimits>> 
+* <<preQuery>> 
+* <<preQueryAll>> 
+* <<preRetrieve>> 
+* <<preSearch>> 
+* <<preUpdate>> 
+* <<updateSobjectCollections>> 
+* <<upsertSobjectCollections>> 
 
 == Operations
 
+[[createSobjectCollections]]
+=== Create SObject Collections
+
+`<salesforce-composite:create-sobject-collections>`
+
+Add up to 200 records and a list of `SaveResult` objects. If an error occurs, 
+you can choose to roll back the entire request.
+
+To roll back a request if a create fails, set the `allOrNone` parameter to `true` in the operation payload:
+
+[source,json,linenums]
+----
+{
+	"allOrNone": true,
+	"sObjects": [sObject1, sObject2, ...]
+}
+----
+
+
+==== Parameters
+
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Request Body a| Object |  The request body |  `#[payload]` | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and to store the expression outcome in the target variable. |  `#[payload]` | 
```

Comment:
Please change all descriptions for Target Value to this:

A variable that stores the output of an expression.

Reply:
This fix is incorrect. The correct fix is:
| Target Variable a| String |  A variable that stores the output of the **Target Variable** expression. |  | 
| Target Value a| String |  An expression to evaluate against the operation's output. Stores the output in the **Target Variable** string. |  `#[payload]` | 

Fixed globally.